services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/repeated_feedback/checker.rb

```diff
@@ -26,8 +28,12 @@ def run
         private def last_attempt? = history.size >= 4
 
         private def system_prompt = Evidence::GenAI::RepeatedFeedback::PromptBuilder.run(prompt: nil, history:)
+        private def entry = HTMLTagRemover.run(feedback.chomp)
+        private def model = chat_api::SMALL_MODEL
```

Comment:
Were larger models giving similar accuracy 82%/100%?

Reply:
I didn't try with Gemini-pro yet, but early on `gpt-4o` and `gpt-4` performed roughly identically to `gpt-4o-mini` (some times slightly better, some times slightly worse). My hypothesis is that for the super short prompt, the size of the model may not matter that much.